src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -194,7 +194,7 @@ public function shutdown()
         $currErrorHandler = set_error_handler('is_int');
         restore_error_handler();
 
-        if ($currErrorHandler !== [$this, 'handleError']) {
+        if ($currErrorHandler !== $this->handleError(...)) {
```

Comment:
Are you sure this works ? To me, this will create a *new* Closure object and so it won't be `===`